perun-ldapc/src/main/resources/perun-ldapc.xml

```diff
@@ -1273,6 +1273,16 @@ http://www.springframework.org/schema/aop http://www.springframework.org/schema/
 						</bean>
 					</property>
 				</bean>
+				<bean class="cz.metacentrum.perun.ldapc.model.impl.PerunAttributeDesc">
```

Comment:
doesn't `perun-schema.ldif` also need to be updated?

Reply:
Please see the updated version.